telegram-bot/src/main/java/com/exadel/telegrambot/bot/utils/Method.java

```diff
@@ -0,0 +1,5 @@
+package com.exadel.telegrambot.bot.utils;
+
+public interface Method {
```

Comment:
Why we need this interface?

Reply:
I can write sandMessage, deleteMessage and so on, in here

Reply:
than we call all methods from there

Reply:
This method not involve in any places, please remove romeve